sdks/python/apache_beam/runners/portability/stager.py

```diff
@@ -935,7 +939,12 @@ def _download_pypi_sdk_package(
           '--platform',
           platform_tag
       ])
-      # Example wheel: apache_beam-2.4.0-cp27-cp27mu-manylinux1_x86_64.whl
+      # Example wheel: apache_beam-2.42.0-cp39-cp39-manylinux1_x86_64.whl
+      # Example wheel: with manylinux14 tag.
+      # apache_beam-2.43.0-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_
```

Comment:
nit: you can move it on the same line for readability and disable linter on this line if necessary